src/runner/linter.ts

```diff
@@ -0,0 +1,172 @@
+import { get } from 'lodash';
+
+import { JsonPath, Optional } from '@stoplight/types';
+import { Document } from '../document';
+import { IMessageVars, message } from '../rulesets/message';
+import { getDiagnosticSeverity } from '../rulesets/severity';
+import { IFunctionResult, IGivenNode } from '../types';
+import { decodeSegmentFragment, getClosestJsonPath, printPath, PrintStyle } from '../utils';
+import { Rule } from '../rule';
+import { IRunnerInternalContext } from './types';
+import { getLintTargets } from './utils/getLintTargets';
+import { IExceptionLocation } from './utils/pivotExceptions';
+
+const arePathsEqual = (one: JsonPath, another: JsonPath): boolean => {
```

Comment:
@P0lip Maybe reuse `/runner/utils/arePathsEqual.ts`?

Reply:
Yeah, seems like some merge leftover.
Thanks for spotting!